drift/info_parser.py

```diff
@@ -218,13 +218,20 @@ def _create_comparison(systems, info_name, reference_id, system_count):
     elif len(system_values) == 1:
         info_comparison = COMPARISON_SAME
 
+    baseline_count = 0
+    for system_id_value in sorted_system_id_values:
+        if system_id_value["is_baseline"]:
+            baseline_count += 1
+
     # override comparison logic for certain fact names
-    if _is_unique_rec_name(info_name) and system_count > 1:
+    if _is_unique_rec_name(info_name) and (system_count - baseline_count) > 1:
         system_values_with_duplicates = [system["value"] for system in system_id_values]
         if len(system_values) == len(system_values_with_duplicates):
             info_comparison = COMPARISON_SAME
         else:
             info_comparison = COMPARISON_DIFFERENT
+        if "N/A" in system_values:
```

Comment:
should this be

```python
if:
  blah
elif "N/A" in system_values:
    info_comparison = COMPARISON_INCOMPLETE_DATA
else:
    info_comparison = COMPARISON_DIFFERENT
```
?

Reply:
I saw this more as an override.  Could do as you say and put incomplete in the first 'if' position.  Is there a reason that is better?

Reply:
only for readability and my personal preference. If you think your way is more readable, go with that. A comment may be helpful in either case

Reply:
Ok, added a comment and shuffled for readability, thanks!